### DISCUSSION TITLE
Is Cardano able to enforce royalties?

### DISCUSSION THREAD

**Discussion by MoistMat** [IntersectMBO/cardano-node#4536] [Q&A]
Solana has a 0% royalties trend, where one marketplace after another started ignoring the royalties set within a collection's metadata.
Are Cardano NFTs able to enforce royalties no matter the marketplace, or is this a universal issue with any blockchain?

**Comment by ltouro** [comment]
I think it is possible to enforce royalties but not that easy and the tooling still needs to grow further. 

Basically, what you need is to lock the NFT (a UTXO) in a Smart Contract that only allows it to be spent (i.e. be transferred to other UTXO) along with some amount of royalties being sent to the NFT creator (i.e. creating new UTXOs for given address).

Note that this makes that such NFT could only exist in a 'locked by SC' state in order to guarantee the royalties. It may render more difficult for wallets to keep track, as the SC will have its own address (possibly multiple).
